Sources/Misc/Signing+ResponseVerification.swift

```diff
@@ -65,6 +65,7 @@ extension HTTPResponse where Body == Data {
     ) -> VerificationResult {
         guard let nonce = request.nonce,
               let publicKey = publicKey,
+              !statusCode.isServerError,
```

Comment:
Should we also filter 4xx? Not sure if that would be caught anywhere else

Reply:
Yup! Let me add a test for that too 👍🏻